cypress.config.ts

```diff
@@ -28,12 +29,20 @@ export default defineConfig({
             return config;
         },
         baseUrl: "http://localhost:3200",
+        downloadsFolder: "cypress/downloads",
```

Comment:
I _think_ you can put shared properties in the 'top level' config, rather than in e2e and component separately

Reply:
Removed it entirely. Cypress docs said it defaults to cypress/downloads.